cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ConstraintsTest.scala

```diff
@@ -136,6 +137,7 @@ class ConstraintsTest extends DocumentingTestBase with SoftReset {
     prepareAndTestQuery(
       title = "Create a node property existence constraint only if it does not already exist",
       text = "If it is unknown if a constraint exists or not but we want to make sure it does, we add the `IF NOT EXISTS`. " +
+        "Note: The `IF [NOT] EXISTS` syntax for constraints is only available in Neo4j 4.1.3 and onwards. " +
```

Comment:
Since this is `CREATE` the `NOT` should not be optional
```suggestion
        "Note: The `IF NOT EXISTS` syntax for constraints is only available in Neo4j 4.1.3 and onwards. " +
```